sdks/python/setup.py

```diff
@@ -104,6 +104,7 @@ def get_version():
 REQUIRED_PACKAGES = [
     'avro>=1.8.1,<2.0.0; python_version < "3.0"',
     'avro-python3>=1.8.1,<2.0.0; python_version >= "3.0"',
+    'backports.weakref>=1.0rc1,<1.1',
```

Comment:
I believe the decision was not to support python 2 for interactive pipelines. Do we need this here?